utils/scripts/util.py

```diff
@@ -59,20 +59,23 @@ def read_yaml(filename):
         return None
 
 
-def execute_command(cmd, get_stderr=False):
+def execute_command(cmd, get_stderr=False, stderrPrint=False):
     """
     Executes command in the local node
     """
     try:
         process = subprocess.run(
             cmd,
             shell=True,
-            check=True,
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
-            universal_newlines=True,
+            text=True
         )
+
         output = process.stdout
+        if stderrPrint and process.stderr:
+            print("ods-ci-Error:{}".format(stderrPrint.stderr, end=""))
```

Comment:
shouldn't it be `process.stderr` instead of `stderrPrint.stderr`?

Reply:
correct  ill replace it thanks :)